help/destinations/catalog/advertising/magnite-streaming.md

```diff
@@ -1,59 +1,57 @@
 
 ### Activation and targeting {#activation-and-targeting}
 
-This integration with Magnite allows customers to pass their CDP audiences from Adobe Experience Platform to Magnite for advertising targeting. Audiences may be selected within Magnite for positive targeting as well as negative targeting (suppression). 
+This integration with [!DNL Magnite] allows customers to pass their CDP audiences from [!DNL Adobe Experience Platform] to [!DNL Magnite], and any [!DNL Magnite] platforms, for advertising targeting. Audiences may be selected within [!DNL Magnite] for positive targeting as well as negative targeting (suppression). 
 
 ## Prerequisites {#prerequisites}
 
-To use the [!DNL Magnite] destinations in Adobe Experience Platform, you must first have a [!DNL Magnite Streaming] account. If you have a [!DNL Magnite Streaming] account, please reach out to your [!DNL Magnite] account manager to be provided credentials to access [!DNL Magnite's] destinations.
-If you do not have a [!DNL Magnite Streaming] account, please reach out to adobe-tech@magnite.com
+To use the [!DNL Magnite] destinations in [!DNL Adobe Experience Platform], you must first have a [!DNL Magnite] account. If you already have a [!DNL Magnite] account, please reach out to your [!DNL Magnite] account manager to be provided credentials to access [!DNL Magnite's] destinations.
+If you do not have a [!DNL Magnite] account, please reach out to adobe-tech@magnite.com
 
 ## Supported identities {#supported-identities}
 
-The [!DNL Magnite: Real-Time] destination supports the activation of identities described in the table below. Learn more about [identities](/help/identity-service/features/namespaces.md).
-
-| Target Identity   | Description                                                                                      | Considerations                                                                       |
-|-------------------|--------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------|
-| device_id         | A unique identifier for a device or identity. We accept any device ID and first-party ID regardless of type.           | Identity types supported by Magnite include but are not limited to PPUID, GAID, IDFA, and TV Device IDs.     | 
-
-{style="table-layout:auto"}
+The [!DNL Magnite: Real-Time] destination supports the activation of any attributes and/or identities stored in [!DNL Adobe Experience Platform], which will need to be mapped to a [!DNL Magnite] supported device type. The list of supported device types is managed by [!DNL Magnite], and may change and/or grow over time. The current list of supported device types will be available from a dropdown in the Target Field, during the Mapping step of Audience Activation (described below). Learn more about [identities](/help/identity-service/features/namespaces.md).
```

Comment:
"The current list of supported device types will be available from a dropdown in the Target Field,".

There is no dropdown when you configure mapping. Please rephrase this part so it reflects the UI.